lib/App/tldr.pm

```diff
@@ -31,12 +33,30 @@ sub parse_options {
         "h|help"    => sub { $self->_help },
         "o|os=s@"   => \($self->{platform}),
         "v|version" => sub { printf "%s %s\n", ref $self, $self->VERSION; exit },
+        "pager=s"   => \my $pager,
+        "no-pager"  => \my $no_pager,
     or exit(2);
     $self->{argv} = \@ARGV;
+    if (!$no_pager and -t STDOUT and my $guess = $self->_guess_pager($pager)) {
+        $self->{pager} = $guess;
+    }
     push @{$self->{platform}}, $self->_guess_platform, "common";
     $self;
 }
 
+sub _guess_pager {
+    my $self = shift;
+
+    my $cmd;
+    for my $try (grep $_, @_, $ENV{PAGER}, "less", "more") {
+        if (my $found = File::Which::which($try)) {
+            $cmd = $found, last;
+        }
+    }
+    return if !$cmd;
+    [$cmd, "-R"];
```

Comment:
`less` works fine for me, but not `more` -

    $ tldr --pager more tar
    more: unknown option -R
    Try 'more --help' for more information.

Perhaps `[$cmd, $cmd eq 'less' ? '-R' : ()]` ?

`tldr tar | more` works fine on my Cygwin x64 - `more` seems to pass the escape sequences unchanged, so colors appear properly.

Reply:
Thanks. Done cdddbe5